### PR TITLE
Added requirements.txt in repository root

### PR DESCRIPTION
**Pull request by noonat**
The docs/requirements.txt, .travis.yml, and readme files were updated to reference this file, so there aren't dependencies specified in multiple places.

@jmathai I'm not sure how you want to handle version pinning. I just pinned to >= the most recent version and < the next major version (hoping for people following semantic versioning). Please let me know if you'd like me to change that.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+docopt>=0.6.2,<1.0
+LatLon>=1.0.2,<2.0
+mock>=1.3.0,<2.0
+nose>=1.3.7,<2.0
```
Comment (arikfr) [inline requirements.txt]: mock and nose are dev dependencies... not sure about including them here.

Comment (jmathai) [inline requirements.txt]: :+1:

**Comment by noonat** [comment]
Moved mock and nose into elodie/tests/requirements.txt.

File: docs/requirements.txt [hunk 2]
```diff
@@ -1,6 +1,3 @@
-LatLon
-docopt
-requests
-mock
-send2trash
-sphinx
+-r ../requirements.txt
```
Comment (jmathai) [inline docs/requirements.txt]: :+1:

**Pull request merged**